src/qt/bitcoingui.cpp

```diff
@@ -1040,6 +1047,41 @@ void BitcoinGUI::snapshotClicked()
     }
 }
 
+void BitcoinGUI::syncfromzeroClicked()
+{
+    QMessageBox Msg;
+
+    Msg.setIcon(QMessageBox::Question);
+    Msg.setText(tr("Do you want to delete blockchain data and sync from zero?"));
+    Msg.setInformativeText(tr("Warning: After the blockchain data is deleted the wallet will shutdown and when restarted will begin syncing from zero. After restart you may continue syncing from zero or use the snapshot download to download the latest snapshot. Your balance will temporarily show as 0 GRC while syncing."));
```

Comment:
I think we want to discourage snapshots moving forward. Should we remove the suggestion to download the snapshot when someone chooses the sync route?

Reply:
I am ok with that I think.

Reply:
done